### ISSUE TITLE
Can I add <script> in <head>?

### ISSUE THREAD

**Issue by stlst** [jharris4/html-webpack-tags-plugin#62]
Now *.js files are added in <body> of *.html by default. Is there an option for me to set <script> tag in <head>?

**Comment by jharris4** [comment]
I'm not sure I understand this question, but the settings for where tags get inserted (head vs body) is controlled by the html-webpack-plugin options.

**Comment by zhipenglin** [comment]
try append:false

**Comment by vinamelody** [comment]
Hi @jharris4 thanks for the pointer, you're right!
currently i have `inject: true` so it's by default, app.bundle.js is added just before closing body tag.

but what if I want certain js to be inserted in head?

**Comment by jharris4** [comment]
It really depends whether you're talking about webpack generated js or js generated by this plugin. There are different settings for each plugin depending on which one you're talking about.

**Comment by vinamelody** [comment]
hmm i'm trying to add an external js. thanks @jharris4 !

**Comment by jharris4** [comment]
Where script tags get inserted is controlled by the settings of `html-webpack-plugin`. This (html-webpack-tags-plugin) plugin only allows controlling the order of external script tags, not their location (head vs body).

You can try changing the `inject` option of the `html-webpack-plugin` to something like `head`, but unfortunately there isn't currently a way to control this on a per-script tag level.

**Comment by vinamelody** [comment]
yup hmm i tried changing it to `head` but then my `app.bundle.js` needs to be placed in body 
for now i'll create a different `index.html` production version to solve this thing first :)
